plugins/include/open62541/plugin/accesscontrol_default.h

```diff
@@ -31,6 +36,11 @@ UA_AccessControl_default(UA_ServerConfig *config,
                          size_t usernamePasswordLoginSize,
                          const UA_UsernamePasswordLogin *usernamePasswordLogin);
 
+UA_EXPORT UA_StatusCode
+UA_AccessControl_setCallback(UA_ServerConfig *config,
```

Comment:
Why not have `UA_AccessControl_defaultWithLoginCallback` in parallel to `UA_AccessControl_default`?
Then we don't have bugs in the interaction of _default and _setCallback.

Reply:
True.  I remember that I had problems in my tests calling them in the correct order.  I will create a diff, test it and push to this pull request if it works better.